Library/Homebrew/tab.rb

```diff
@@ -92,7 +92,7 @@ def self.from_file_content(content, path)
     end
 
     if attributes["source"]["spec"].nil?
-      version = PkgVersion.parse path.to_s.split("/").second_to_last
+      version = PkgVersion.parse path.to_s.split("/")[-2]
```

Comment:
`File.basename(File.dirname(path))` might be a lot clearer what's going on here.

Reply:
Or rely on `Pathname` and use `parent`

Reply:
Good call, done